egs2/wsj0_2mix_spatialized/enh1/local/data.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
```

Comment:
ditto

Reply:
I'm considering merging this one and local/data.sh in egs2/wsj0_2mix.

Reply:
I'm considering merging this one and local/data.sh in egs2/wsj0_2mix.